<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoParoley</class>
 <widget class="QDialog" name="OknoParoley">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="spis_paroley">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="pass_edit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="notice_edit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>411</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dobav_knopka">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prim_knopka">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>410</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="izm_knopka">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>410</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_knopka">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
